plugins/generic/pln/classes/DepositDAO.inc.php

```diff
@@ -124,7 +124,7 @@ function &getNeedPackaging($journalId) {
 			array(
 				(int) $journalId,
 				(int) PLN_PLUGIN_DEPOSIT_STATUS_PACKAGED,
-				(int) PLN_PLUGIN_DEPOSIT_STATUS_LOCAL_FAILURE
+				(int) PLN_PLUGIN_DEPOSIT_STATUS_LOCKSS_AGREEMENT
```

Comment:
There is only one journal in the wild that has deposits in the pln_deposits table. We'll ask them to reset the deposits so they get regenerated.